config.json

```diff
@@ -232,6 +232,12 @@
             "Project": "signing\\sign.proj"
           }
         },
+        "test": {
+          "description": "Build and run tests. Assumes product packages are already built.",
```

Comment:
So this means "Build tests then run tests.  Assumes product packages are already built." I was confused initially.

Reply:
Switched to your wording.